.goreleaser.yaml

```diff
@@ -35,12 +35,16 @@ builds:
       - goos: windows
         goarch: arm64
 docker_manifests:
-  - name_template: quay.io/jpkroehling/otelcol:{{ .Tag }}
+  - name_template: quay.io/opentelemetry/otelcol:{{ .Tag }}
```

Comment:
Why this? Who owns this?

Reply:
Legacy. We created this when we didn't know who owned DockerHub's "opentelemetry" namespace. Later on, "otel" was created there and we stopped publishing to Quay, IIRC.

Reply:
And these people have access to the repository: 

![image](https://user-images.githubusercontent.com/13387/132339573-ff57b0c7-5d31-4412-aa83-4a65275c960e.png)

I should probably remove @objectiser and @pavolloffay from it, though, as they are not maintainers of this project.

Reply:
I would like to see OTEL starting publishing to `ghcr.io` than quay or docker hub.